core/src/Kokkos_CopyViews.hpp

```diff
@@ -94,6 +94,7 @@ struct ViewFill<ViewType, Layout, ExecSpace, 1, iType, ETIAvail> {
   typename ViewType::const_value_type val;
   typedef typename ViewType::non_const_value_type ST;
   ViewFill(const ViewType&, const ST&);
```

Comment:
Don't you want to replace this constructor similar to the other places?

Reply:
Jup that one just needs to be deleted (its not implemented further down ...